### PR TITLE
Fix copyright and Update package.json

### PR DESCRIPTION
**Pull request by PuneetGopinath**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [ ] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [x] **Other:** `Update copyright holders` 

## Description

You have to keep the copyright holder as `The Readme-Workflows organisation` or `The Readme-Workflows Team` instead of `Abishek Joshi, Puneet Gopinath`.

Closes #

[contributing]: https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Copyright (c) 2020 James George
+ * Copyright (c) 2021 The Readme-Workflows organisation
```
Comment (abhijoshi2k) [inline config.js:3]:
```suggestion
 * Copyright (c) 2021 The Readme-Workflows organization owners
```

**Review by abhijoshi2k — CHANGES_REQUESTED** [review]

File: config.js (hunk 1 above)
Comment (Andre601) [reply config.js:3]: Maybe `Copyright (c) 2021 Readme-Workflows Contributors`?

Comment (PuneetGopinath) [reply config.js:3]: I don't think we need owners as it is understandable

Comment (abhijoshi2k) [reply config.js:3]: Yes agreed. Contributors.

Comment (PuneetGopinath) [reply config.js:3]: I don't think we need owners as it is understandable

Comment (PuneetGopinath) [reply config.js:3]: Do you actually mean and contributors

Comment (PuneetGopinath) [reply config.js:3]: As we are not contributors, we are members or collaborators of this org.

Comment (abhijoshi2k) [reply config.js:3]: Yeah we can add `and contributors`

Comment (PuneetGopinath) [reply config.js:3]: Is anyone online?

**Comment by PuneetGopinath** [comment]
@Andre601 @abhijoshi2k Plz review

File: LICENSE [hunk 2]
```diff
@@ -1,7 +1,7 @@
 MIT License
 
 Copyright (c) 2020 James George
-Copyright (c) 2021 Abhishek Joshi and Puneet Gopinath
+Copyright (c) 2021 The Readme-Workflows organisation
```
Comment (Andre601) [inline LICENSE:4]: Misses the "and Contributors"

**Review by abhijoshi2k — APPROVED** [review]

**Review by Andre601 — CHANGES_REQUESTED** [review]

**Comment by PuneetGopinath** [comment]
@Andre601 Are you busy to review?

**Comment by PuneetGopinath** [comment]
@Andre601 Plz review

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Can you check out messages in discord

**Review by Andre601 — APPROVED** [review]

**Pull request merged**